core/domain/collection_domain.py

```diff
@@ -499,43 +638,99 @@ def is_demo(self):
         return self.is_demo_collection_id(self.id)
 
     def update_title(self, title):
+        """Updates the title of the collection.
+
+        Args:
+            title: str. The new title for the collection which is to be updated.
+        """
         self.title = title
 
     def update_category(self, category):
+        """Updates the category of the collection.
+
+        Args:
+            category: str. The category of the collection which is to be
+                updated.
+        """
         self.category = category
 
     def update_objective(self, objective):
+        """Updates the objective of the collection.
+
+        Args:
+            title: str. The objective of the collection which is to be updated.
+        """
         self.objective = objective
 
     def update_language_code(self, language_code):
+        """Updates the language code of the collection.
+
+        Args:
+            title: str. The language code of the collection which is to be
+                updated.
+        """
         self.language_code = language_code
 
     def update_tags(self, tags):
+        """Updates the tags of the collection.
+
+        Args:
+            title: list. The tags of the collection which is to be updated.
+        """
         self.tags = tags
 
     def _find_node(self, exploration_id):
-        for ind, node in enumerate(self.nodes):
+        """Returns the index of the collection node with the given exploration
+        id and None if the exploration id is not in the nodes list.
+
+        Args:
+            exploration_id: str. The id of the exploration.
+
+        Returns:
+            int. if the list of nodes contain the given exploration id
+                then it will return the index of that node, else None.
+        """
+        for index, node in enumerate(self.nodes):
             if node.exploration_id == exploration_id:
-                return ind
+                return index
         return None
 
     def get_node(self, exploration_id):
         """Retrieves a collection node from the collection based on an
         exploration ID.
+
+        Args:
+            exploration_id: str. The id of the exploration.
+
+        Returns:
+            ColectionNode. if the list of nodes contain the given exploration
```

Comment:
Spelling of CollectionNode is wrong.

if --> If

contain --> contains (since it refers to the list)